src/components/Captures/CaptureTable.js

```diff
@@ -0,0 +1,225 @@
+import CaptureDetailDialog from '../CaptureDetailDialog';
+import { tokenizationStates } from '../../common/variables';
+import useStyle from './CaptureTable.styles.js';
+
+const columns = [
+  {
+    attr: 'id',
+    label: 'Capture ID',
+  },
+  {
+    attr: 'planterId',
+    label: 'Grower ID',
+  },
+  {
+    attr: 'deviceIdentifier',
+    label: 'Device Identifier',
+    noSort: false,
+  },
+  {
+    attr: 'planterIdentifier',
+    label: 'Planter Identifier',
+    noSort: false,
+  },
+  {
+    attr: 'verificationStatus',
+    label: 'Verification Status',
+    noSort: true,
+  },
+  {
+    attr: 'speciesId',
+    label: 'Species',
+    noSort: true,
+  },
+  {
+    attr: 'tokenId',
+    label: 'Token Status',
+    renderer: (val) =>
+      val ? tokenizationStates.TOKENIZED : tokenizationStates.NOT_TOKENIZED,
+  },
+  {
+    attr: 'timeCreated',
+    label: 'Created',
+    renderer: (val) => getDateTimeStringLocale(val),
+  },
+];
+
+const CaptureTable = () => {
+  const {
+    filter,
+    rowsPerPage,
+    page,
+    order,
+    orderBy,
+    captures,
+    capture,
+    captureCount,
+    setPage,
+    setRowsPerPage,
+    setOrder,
+    setOrderBy,
+    getCaptureAsync,
+  } = useContext(CapturesContext);
+  const speciesContext = useContext(SpeciesContext);
+  const [isDetailsPaneOpen, setIsDetailsPaneOpen] = useState(false);
+  const [speciesState, setSpeciesState] = useState({});
+  const scrollRef = createRef();
+  const classes = useStyle();
+
+  useEffect(() => {
+    formatSpeciesData();
+  }, [filter]);
+
+  const formatSpeciesData = async () => {
+    let species = {};
+    speciesContext.speciesList.map((s) => {
+      species[s.id] = s.name;
+    });
+    setSpeciesState(species);
+  };
+
+  const toggleDrawer = (id) => {
+    getCaptureAsync(id);
+    setIsDetailsPaneOpen(!isDetailsPaneOpen);
+  };
+
+  const createToggleDrawerHandler = (id) => {
+    return () => {
+      toggleDrawer(id);
+    };
+  };
+
+  const closeDrawer = () => {
+    setIsDetailsPaneOpen(false);
+  };
+
+  const handlePageChange = (e, page) => {
+    setPage(page);
+  };
+
+  const handleRowsPerPageChange = (e) => {
+    setRowsPerPage(parseInt(e.target.value));
+  };
+
+  const createSortHandler = (attr) => {
+    return () => {
+      const order = orderBy === attr && order === 'asc' ? 'desc' : 'asc';
+      const orderBy = attr;
+      setOrder(order);
+      setOrderBy(orderBy);
+    };
+  };
+
+  const tablePagination = () => {
+    return (
+      <TablePagination
+        rowsPerPageOptions={[25, 50, 100, 250, 500]}
+        component="div"
+        count={captureCount || 0}
+        page={page}
+        rowsPerPage={rowsPerPage}
+        onChangePage={handlePageChange}
+        onChangeRowsPerPage={handleRowsPerPageChange}
+      />
+    );
+  };
+
+  return (
+    <Grid item container style={{ height: '100%', overflow: 'auto' }}>
+      <div className={classes.tableContainer} ref={scrollRef}>
```

Comment:
This container isn't filling the width of the window.

Reply:
Thanks for catching that!